tests/unit/use-event-data.test.js

```diff
@@ -41,6 +41,7 @@ const timezonedEvents = [
 ];
 it('should remove events that have past', () => {
     const filtered = removePastEvents(events);
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Should we just not assign a variable below?

Reply:
Fair point